app/deed/search/__init__.py

```diff
@@ -0,0 +1,6 @@
+from flask import Blueprint
+from . import routes, server
+
+blueprint = Blueprint('search', __name__)
```

Comment:
Not sure about the answer here, just a question, should we be naming the blueprints based on the hierarchy? i.e. this is the deed search bit so `deed.search`?